app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,94 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+
+/* typography */
+
+h1, h2, h3, h4, h5, h6 {
+  line-height: 1;
+}
+
+h1 {
+  font-size: 3em;
+  letter-spacing: -2px;
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.2em;
```

Comment:
Properties should be ordered color, font-size, font-weight, letter-spacing, margin-bottom, text-align